milli/src/update/index_documents/typed_chunk.rs

```diff
@@ -176,6 +178,22 @@ pub(crate) fn write_typed_chunk_into_index(
             )?;
             is_merged_database = true;
         }
+        TypedChunk::GeoPoints(mut geo_points) => {
+            // TODO: TAMO: we should create the rtree with the `RTree::bulk_load` function
+            let mut rtree = index.geo_rtree(&index.read_txn()?)?.unwrap_or_default();
```

Comment:
```suggestion
            let mut rtree = index.geo_rtree(&wtxn)?.unwrap_or_default();
```

You must always read your database with thee same, with the same transaction. use the `wtxn` given to you.